test/aws/osml/models/centerpoint/test_centerpoint_model.py

```diff
@@ -56,4 +56,4 @@ def test_predict_bad_data_file(self):
         data_binary = None
         response = self.client.post("/invocations", data=data_binary)
 
-        assert response.status_code == 400
+        assert response.status_code == 500
```

Comment:
Why did this change? If the client sends a bad request it should get a 4xx client error not a 5xx server error.

Reply:
Will update